apps/voting/app/src/useScrollTop.js

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react'
+
+function useScrollTop(dependency) {
+  useEffect(() => {
+    window.scrollTo(0, 0)
```

Comment:
Just as a note: this won’t work if we decide to move to a local scroll (for `SidePanel`), or if we want to reuse it in other cases (e.g. internal apps in aragon/aragon).

Another option is to use `scrollIntoView()` on an element, and let the browser do the rest: https://github.com/aragon/aragon/blob/master/src/apps/Permissions/Permissions.js#L224-L228

Reply:
> this won’t work if we decide to move to a local scroll (for SidePanel)

Is that something that has been decided? It currently works as is. I'm fond of the `yagni` idea, if/when the change occurs then we should update this.

Reply:
Sounds good :+1:  I posted this so we remember that an update might be needed.

Reply:
Perf, thanks 😄